### DISCUSSION TITLE
(UX Feature Request) Make the dropdown menu searchable for menus with many options

### DISCUSSION THREAD

**Discussion by NiC0x36** [iotaledger/firefly#1541] [Features / Enhancements]
## Description

It's a bit hard to search for the desired language and currency under General Settings as the lists of the dropdown menus are very long.

![image](https://user-images.githubusercontent.com/15385922/111000085-b70c7980-8381-11eb-9b97-9d5c2a69e013.png)

## Motivation

Better user experience, easier & faster to find the desired language and currency, less frustration. Smart feature that the user will appreciate.

## Requirements

Write a list of what you want this feature to do.

1. Implement a searchfield in the dropdown menu similar to this:
![image](https://user-images.githubusercontent.com/15385922/110999792-3a799b00-8381-11eb-8fe2-88b6d523e9b5.png)
![image](https://user-images.githubusercontent.com/15385922/110999815-436a6c80-8381-11eb-9bf0-63e4d04914db.png)

## Are you planning to do it yourself in a pull request?

No.

**Comment by cvarley100** [comment]
@digital-zen thoughts?
